apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -70,6 +77,7 @@ const Page = async ({ params }: { params: Params }) => {
       urlReplaces: urlReplacesMap,
     },
   };
+  const t = useTranslations('shared');
```

Comment:
I see this error:
![image](https://github.com/pagopa/developer-portal/assets/39835990/e17e3024-230e-4506-a4ec-1934e69961b5)
To avoid it you could mark the component with the `use client;` directive, but the export of "generateMetadata" from a client component is disallowed and you would have another error.

I think you should move `useTranslations` to the `RelatedLinks` component and use `t('relatedLinks')` as the fallback value of the title property.